recipes/ftjam/all/conanfile.py

```diff
@@ -0,0 +1,128 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class FtjamConan(ConanFile):
+    name = "ftjam"
+    description = "Jam (ftjam) is a small open-source build tool that can be used as a replacement for Make."
+    topics = ("conan", "ftjam", "build", "make")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.freetype.org/jam/"
+    license = "BSD-3-Clause"
+    exports_sources = "patches/*"
+    settings = "os", "arch", "compiler", "build_type"
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if self.settings.compiler == "Visual Studio":
+            raise ConanInvalidConfiguration("ftjam does not work, built with Visual Studio")
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("ftjam-{}".format(self.version), self._source_subfolder)
+
+    def build_requirements(self):
+        if tools.os_info.is_windows and "CONAN_BASH_PATH" not in os.environ and \
+                tools.os_info.detect_windows_subsystem() != "msys2":
+            self.build_requires("msys2/20190524")
+        if self.settings.compiler == "Visual Studio":
+            self.build_requires("automake/1.16.2")
+        if self.settings.os != "Windows":
+            self.build_requires("bison/3.5.3")
+
+    def _patch_sources(self):
+        tools.replace_in_file(os.path.join(self._source_subfolder, "jamgram.c"),
+                              "\n#line", "\n//#line")
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+        self._autotools.libs = []
+        # The configure MUST be run inside this directory
+        with tools.chdir(os.path.join(self.build_folder, self._source_subfolder, "builds", "unix")):
+            self._autotools.configure()
+        return self._autotools
+
+    def build(self):
+        if self.settings.build_type != "Release":
```

Comment:
Is there any reason for using this under build() and not configure() ?

Reply:
I did it for the same reason as in the recent ninja update by eric

Reply:
Agreed!